web/generated_assets.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !genassets
+
+package web
+
+var htmlContent = `<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>{{.Name}}</title>
+        <style>{{.CSS}}</style>
+    </head>
+    <body>
+        <h1>{{.Name}}{{if .Version}} <span class="version">{{.Version}}</span>{{end}}</h1>
+        <nav><ul>
+            {{ range .Links }}
+            <li><a href="{{ .Address }}">{{.Text}}</a>{{ if .Description }}: {{.Description}}{{end}}</li>
+            {{ end }}
+            </ul>
+        </nav>
+    </body>
+</html>
+`
+var cssContent = `Hello
+`
```

Comment:
How about something simple to start.

```suggestion
var cssContent = `
h1 {
   color: #fff;
   background-color: #343a40
}
`
```